2-datatypes/boolean.js

```diff
@@ -0,0 +1,16 @@
+const prompt = require("prompt");
+
+prompt.start();
+
+prompt.get(["input"], main)
+function main(err, res) {
+    let parsedinput = false;
+    if (res.input === "true") {
+        console.log(parsedinput = true);
```

Comment:
Do not do assignment and logging at same line. Do `parsedinput = true` here.

`console.log(parsedinput` at the end of functions.

Reply:
updated codes